beacon-chain/monitor/process_block.go

```diff
@@ -0,0 +1,46 @@
+package monitor
+
+import (
+	"fmt"
+
+	types "github.com/prysmaticlabs/eth2-types"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/blocks"
+	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
+	"github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/block"
+	"github.com/sirupsen/logrus"
+)
+
+// processSlashings logs the event of one of our tracked validators was slashed
+func (s *Service) processSlashings(blk block.BeaconBlock) {
+	for _, slashing := range blk.Body().ProposerSlashings() {
+		idx := slashing.Header_1.Header.ProposerIndex
+		if s.TrackedIndex(idx) {
+			log.WithFields(logrus.Fields{
+				"ProposerIndex":  idx,
+				"SlashedInSlot:": blk.Slot(),
+				"Slot":           slashing.Header_1.Header.Slot,
```

Comment:
```suggestion
				"HeadSlot":           slashing.Header_1.Header.Slot,
```
then add a `CurrentSlot`